devops/azure-pipelines.yml

```diff
@@ -76,8 +76,55 @@ stages:
         containerServiceConnection: 'acr-team1'
         discordChannel: $(discordFrontendChannel)
         discordKey: $(discordFrontendKey)
-
         workDir: "./"
 
     displayName: Dockerize frontend --> Job
   displayName: Dockerize frontend --> Stage 
+
+- stage: deploy
```

Comment:
put a condition here that doesn't consider deploying if it's a pullrequest build. i think if there was a PR from dev, sourcebranch might still say dev.

Reply:
Oops, I have this on the Gateway and forgot to copy it to the others.

Reply:
i'll just remove my request for review from the others then, so re-add it when fixed.

Reply:
Okay, I got them fixed.